packages/cli/generators/controller/index.js

```diff
@@ -235,6 +235,7 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
         'Controller %s is now created in src/controllers/',
         this.artifactInfo.name,
       );
+      super.updateIndexFile();
```

Comment:
Should this be returned since it's a promise? It might not matter since it's the last thing being done here, but I think it makes sense to return it for the sake of using a promise

Reply:
Also, shouldn't `updateIndexFile` be called in `ArtifactGenerator` since most artifacts seem to use `index.ts`?